src/app/multiple-records/multiple-records.component.html

```diff
@@ -15,7 +15,7 @@ <h1 class="govuk-heading-xl govuk-!-margin-bottom-5">
     <ng-container
       *ngFor="let vTechRecord of vehicleTechRecords | FilterMultipleRecords; let i = index"
     >
-      <ng-container *ngIf="vTechRecord.techRecord[0] as activeRecord">
+      <ng-container *ngIf="vTechRecord.techRecord[1] as activeRecord">
```

Comment:
Hi Irina, Please let catch up on this. Just think if it is a future proof way of grabbing what you want